Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -242,8 +258,14 @@ void EnhancementsWidget::ConnectWidgets()
     m_block_save = true;
     m_configure_color_correction->setEnabled(g_Config.backend_info.bSupportsPostProcessing);
     LoadPPShaders();
-    m_block_save = false;
 
+    auto current_stereo_mode = Config::GetUncached(Config::GFX_STEREO_MODE);
```

Comment:
Is there a reason why you can't use the regular `Config::Get`? `Config::GetUncached` should only really be needed internally in `Config::Get`.